doc-content/enterprise-only/installation/clustering-kie-server-failover-proc.adoc

```diff
@@ -0,0 +1,106 @@
++
+[source]
+----
+<cache-container name="jbpm">
+  <transport lock-timeout="60000"/>
+  <replicated-cache name="nodes">
+      <transaction mode="BATCH"/>
+  </replicated-cache>
+  <replicated-cache name="jobs">
+      <transaction mode="BATCH"/>
+  </replicated-cache>
+</cache-container>
+----
+.. Add the `jgroups` subsystem for the replicated and distributed caches:
++
+[source]
+----
+<subsystem xmlns="urn:jboss:domain:jgroups:7.0">
+    <channels default="ee">
+        <channel name="ee" stack="udp" cluster="ejb"/>
+    </channels>
+    <stacks>
+        <stack name="udp">
+            <transport type="UDP" socket-binding="jgroups-udp"/>
+            <protocol type="PING"/>
+            <protocol type="MERGE3"/>
+            <socket-protocol type="FD_SOCK" socket-binding="jgroups-udp-fd"/>
+            <protocol type="FD_ALL"/>
+            <protocol type="VERIFY_SUSPECT"/>
+            <protocol type="pbcast.NAKACK2"/>
+            <protocol type="UNICAST3"/>
+            <protocol type="pbcast.STABLE"/>
+            <protocol type="pbcast.GMS"/>
+            <protocol type="UFC"/>
+            <protocol type="MFC"/>
+            <protocol type="FRAG3"/>
+          </stack>
+          <stack name="tcp">
+            <transport type="TCP" socket-binding="jgroups-tcp"/>
+            <socket-protocol type="MPING" socket-binding="jgroups-mping"/>
+            <protocol type="MERGE3"/>
+            <socket-protocol type="FD_SOCK" socket-binding="jgroups-tcp-fd"/>
+            <protocol type="FD_ALL"/>
+            <protocol type="VERIFY_SUSPECT"/>
+            <protocol type="pbcast.NAKACK2"/>
+            <protocol type="UNICAST3"/>
+            <protocol type="pbcast.STABLE"/>
+            <protocol type="pbcast.GMS"/>
+            <protocol type="MFC"/>
+            <protocol type="FRAG3"/>
+          </stack>
+      </stacks>
+    </subsystem>
+----
+.. Configure the following interface:
++
+[source]
+----
+<interface name="private">
+    <inet-address value="${jboss.bind.address.private:127.0.0.1}"/>
+</interface>
+----
+.. Configure the following socket:
++
+[source]
+----
+<socket-binding-group name="standard-sockets" default-interface="public" port-offset="${jboss.socket.binding.port-offset:0}">
+
+<socket-binding name="jgroups-mping" interface="private" multicast-address="${jboss.default.multicast.address:230.0.0.4}" multicast-port="45700"/>
+
+  <socket-binding name="jgroups-tcp" interface="private" port="7600"/>
+  <socket-binding name="jgroups-tcp-fd" interface="private" port="57600"/>
+  <socket-binding name="jgroups-udp" interface="private" port="55200" multicast-address="${jboss.default.multicast.address:230.0.0.4}" multicast-port="45688"/>
+  <socket-binding name="jgroups-udp-fd" interface="private" port="54200"/>
+----
+. Install the Infinispan extension:
+.. Open the `kie-server.war/WEB-INF/jboss-deployment-structure.xml` file in a text editor and add the `jboss-deployment-structure.xml` below the following dependencies:
```

Comment:
Can you add an example of how the jboss-deployment-structure.xml will look?

Reply:
@antonio fernandez alhambra can you help with an example?

Reply:
Thank you @afalhambra . I don't think we need to include the example in the doc.